### ISSUE TITLE
Histogram library

### ISSUE THREAD

**Issue by twood02** [gwAdvNet2015/adv-net-samples#8]
Write a library in `util/hist.c` to help generate statistics for large numbers of data readings.

Issue #6 builds a Latency Timer program that measures the round trip delay between a client a server. To be accurate, you want to gather hundreds or thousands of such measurements, but if you just report the average that doesn't give the complete picture. This histogram library will help by providing an easy way to summarize large amounts of data.  

The interface should be something like:

```
// put code in util/hist.h and util/hist.c

/* Create an array with nun_buckets that will store data from min to max */
struct histogram* histogram_create(int min, int max, int num_buckets);

/* Increment the bucket for data_point */
void histogram_inc(struct histogram* hist, int data_point);

/* Print out the histogram bucket counts */
void histogram_print(struct histogram* hist);
```

You should use uint32 or uint64 types instead of ints!

**Comment by akorzan** [comment]
I would like to claim this issue for the next two days.

Sincerely,
Anthony

**Comment by twood02** [comment]
closed by #91
